### ISSUE TITLE
Don't link clips to vtubers in certain cases

### ISSUE THREAD

**Issue by Type-kun** [RiceCakess/Holodex#150]
There's a certain problem with Omaru Polka's clip page at https://holodex.net/channel/UCK9V2B22uJYu3N7eR_BT9QA/clips

It's currently dominated by unrelated clips from Hololive Cut channel, which uses Polka's outro as their own clip outro and links Polka's channel in _every clip_ because of that. Blacklisting the subber kind of works, but that's per-user, and results in nearly-empty pages for polka's clips, since the filtering is done on client side after pagination. Moreover, it blacklists the subber globally, hiding legit clips on other vtuber channels.

It would help if there was some kind of internal strategy/option to amend the clip attachment logic for certain subbers. In this case, if the clip is from "Hololive Cut", and only Polka's channel link is specified (not Polka's stream), then it shouldn't be linked to her clips page.

**Comment by theultramage** [comment]
I noticed this sort of behavior too. There's a channel that puts in a variety of "other recommended clips", which just adds noise but is mostly limited to this clipper's videos. Your example is much more severe, as it actually interferes with primary functionality of the website. Mentioning further examples, the talents are also doing this. I've noticed that a lot of them use the same boilerplate description everywhere, sometimes unchanged for months/years. For example, Mori Calliope's links to each holoEN's debut stream.

What I suggest is, for each channel (talent or clipper), have a collection of search patterns that identify the start of this channel's "boilerplate" section of the description, and ignore everything starting from there. That's my quick&dirty idea.

Implementing this sort of solution would require a complete reset and rebuild of all the links. I have been told that the holodex crawler never deletes links, only adds new ones. So links orphaned after a description change, and changes to links done by hand, would get lost. If manual link editing is ever implemented, it needs to consider the possibility of these resets.
